.gitignore

```diff
@@ -1,2 +1,3 @@
+.vscode
```

Comment:
Sorry, I gave a wrong information about this yesterday :(

Reply:
That's fine. Let's add `.vscode/` in `.gitignore`.